redwood-compose/src/commonTest/kotlin/app/cash/redwood/compose/ChangeListenerTest.kt

```diff
@@ -56,7 +56,7 @@ class ProtocolChangeListenerTest : AbstractChangeListenerTest() {
     snapshot: () -> T,
   ): TestRedwoodComposition<T> {
     val composeBridge = TestSchemaProtocolBridge.create()
-    val widgetBridge = ProtocolBridge(MutableListChildren(), TestSchemaProtocolNodeFactory(factories)) {
+    val widgetBridge = ProtocolBridge(MutableListChildren(), TestSchemaProtocolFactory(factories)) {
```

Comment:
The protocol traditionally used "compose" and "widget" naming, but in practice it's "guest" and "host", respectively. I'd like to do a rename of modules and types in a future change.